std/dynamic_library.zig

```diff
@@ -1,10 +1,19 @@
-const std = @import("index.zig");
+const builtin = @import("builtin");
+const Os = builtin.Os;
+const std = @import("std");
```

Comment:
From within the standard library, you have to refer to the standard library by the path to the main file. How it was before was correct.

Reply:
sorry, i was having issue with git so i moved it out of the std directory to work on it before i moved it back.